redhat-developer/developers.redhat.com
# [DEVELOPER-5411] Update iFramely API key to RHD account key.

### JIRA Issue Link
https://issues.jboss.org/browse/DEVELOPER-5411

### Verification Process
- Add Embedded Content
- View that it embeds correctly
- View source of <iframe> and confirm "key" url parameter is set to "c3cf64c36c378b376d8984711011e196"
